core/src/utils/prove.rs

```diff
@@ -217,6 +219,23 @@ where
             .collect::<Vec<_>>();
         shard_proofs.append(&mut checkpoint_proofs);
     }
+    // Log some of the `ExecutionReport` information.
+    tracing::info!(
+        "execution report (totals): instructions={}, syscalls={}",
+        report_aggregate.total_instruction_count(),
+        report_aggregate.total_syscall_count()
+    );
+    tracing::info!("execution report (syscall counts):");
```

Comment:
i wonder if there's a way you can do this that's more nice using tables or something? or fixed padding keys

":>12" type of formatting